### PR TITLE
add signxml

### PR DESCRIPTION
**Pull request by fhoehle**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vend other packages
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by fhoehle** [comment]
@conda-forge-admin, please add noarch: python

File: recipes/signxml/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    - python
+
+test:
+  imports:
+    - signxml
+    - signxml.util
+
+about:
+  home: "https://github.com/kislyuk/signxml"
+  license: "Apache Software"
```
Comment (marcelotrevisani) [inline recipes/signxml/meta.yaml]:
```suggestion
  license: "Apache-2.0"
```

File: recipes/signxml/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,54 @@
+{% set name = "signxml" %}
+{% set version = "2.6.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "70e3edbb07b89bec94d39db2cdced724c540f9258366474177c746b9f903d9c4"
```
Comment (marcelotrevisani) [inline recipes/signxml/meta.yaml]: Could you please remove the quotes from ``sha256``?
```suggestion
  sha256: 70e3edbb07b89bec94d39db2cdced724c540f9258366474177c746b9f903d9c4
```

File: recipes/signxml/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
```
Comment (marcelotrevisani) [inline recipes/signxml/meta.yaml]:
```suggestion
  script: "{{ PYTHON }} -m pip install . -vv"
```

File: recipes/signxml/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+  host:
+    - asn1crypto >=0.24.0
+    - certifi >=2018.1.18
+    - cryptography >=2.1.4
+    - defusedxml >=0.5.0
+    - eight >=0.4.2
+    - lxml >=4.2.1
+    - pip
+    - pyopenssl >=17.5.0
+    - python
```
Comment (marcelotrevisani) [inline recipes/signxml/meta.yaml]: I think here you just need ``python`` and ``pip``

File: recipes/signxml/meta.yaml (hunk 3 above)
Comment (fhoehle) [reply recipes/signxml/meta.yaml]: why dropping `--no-deps`? Without changing requirements will not be catched by the import tests.

Comment (fhoehle) [reply recipes/signxml/meta.yaml]: @marcelotrevisani Why dropping --no-deps? Without changing requirements will not be catched by the import tests.

Comment (marcelotrevisani) [reply recipes/signxml/meta.yaml]: ``conda-build`` already handles it

Comment (fhoehle) [reply recipes/signxml/meta.yaml]: okay, thanks!

**Comment by fhoehle** [comment]
@marcelotrevisani thanks for the review, all comments were addressed.

**Comment by scopatz** [comment]
Thanks!

**Pull request merged**